public/apps/configuration/panels/role-view/index-permission-panel.tsx

```diff
@@ -132,7 +132,15 @@ function getColumns(
           return EMPTY_FIELD_VALUE;
         }
 
-        return renderExpression('Document-level security', JSON.parse(dls));
+        // TODO: unify the experience for both cases which may require refactoring of renderExpression.
+        try {
+          return renderExpression('Document-level security', JSON.parse(dls));
+        } catch (e) {
+          // support the use case for $expression in DLS.
```

Comment:
We could make it more clear like: 
Support the use case for $variable without double quotes in DLS, e.g. variable is an array.

Reply:
Sure. Updated.